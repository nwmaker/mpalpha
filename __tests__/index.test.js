/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import App from '../pages/index.js'

describe('With Enzyme', () => {
  it('App shows "Welcome to MPost"', () => {
    const app = shallow(<App />)

    expect(app.find('h1').text()).toEqual('Welcome to MPost')
  })
})

describe('With Snapshot Testing', () => {
  it('App shows "Welcome to MPost"', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
