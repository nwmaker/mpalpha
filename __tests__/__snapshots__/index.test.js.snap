// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Welcome to MPost" 1`] = `
<div
  className="jsx-3868048425 top-container"
>
  <h1
    className="jsx-3868048425"
  >
    Welcome to MPost
  </h1>
</div>
`;
